tests/test_actions.py

```diff
@@ -247,7 +247,11 @@ def testFractionalDifference(self):
 
     # Basic vectorActions
 
-    # def testLoadVector(self): TODO: implement
+    def testLoadVector(self):
+        action = LoadVector(vectorKey="vector")
+        inputData = {"vector": np.arange(5)}
```

Comment:
```suggestion
        inputData = {"vector": [0, 1, 2, 3, 4]}
```
Just to illustrate that you can make (or may receive) suggestions on code reviews. I generally incorporate them into existing commits through an interactive rebase, as [recommended in the dev guide](https://developer.lsst.io/work/flow.html#pushing-code).